theme/home.less

```
@@ -102,6 +102,11 @@
                 border-color: @heroBannerDotOutlineAtiveColor;
                 background-color: transparent;
             }
+
+            &:focus {
```

This should avoid having the style applied when a user clicks.

```suggestion
            &:focus-visible {
```